### PR TITLE
Feature/add live photos

### PR DESCRIPTION
**Pull request by hjheath**
@Alfreddd Please review

### CODE REVIEW

**Comment by hjheath** [comment]
Build failing because flake8 is no longer supported on python2.6

https://gitlab.com/pycqa/flake8/issues/187

File: onfido/live_photos.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"""Wrapper for the live photos endpoint."""
+import mimetypes
+
+from. api_resource import ApiResource
```
Comment (peterroelants) [inline onfido/live_photos.py]: Why is there a dot after `from`? Is this supposed to be a relative import? In this case move the dot to before `api_resource`, and add a space.

File: onfido/live_photos.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    def create(self, applicant_id, live_photo, filename):
+        """POST to /live_photos
+
+        Args:
+            applicant_id: ID of the applicant this photo belongs to (string).
+            live_photo: The live photo to upload (file-like object).
+            filename: The name of the live photo file (string).
+        """
+        content_type = mimetypes.guess_type(filename)
+        file = {'file': (filename, live_photo, content_type[0])}
```
Comment (peterroelants) [inline onfido/live_photos.py]: don't use `file` as a variable name, it overwrites the the buildin file method (at least in Python 2)

File: onfido/onfido.py [hunk 3]
```diff
@@ -18,3 +19,4 @@ def __init__(self, api_key, version='v2', requestor=None):
         self.Checks = Checks(self.requestor)
```
Comment (peterroelants) [inline onfido/onfido.py:19]: CapitalCase variables? Most of the variables I've seen here are snake_case.

File: onfido/test/test_integration.py [hunk 4]
```diff
@@ -57,6 +57,14 @@ def test_integration_document(self):
 
         self.assertIn("id", doc_response)
 
+    def test_integration_live_photo(self):
+        api = self.create_api()
+        applicant = random.choice(api.Applicants.all()["applicants"])
```
Comment (peterroelants) [inline onfido/test/test_integration.py]: Using random functions in tests? Don't we want them to be reproducible all the time?

File: onfido/live_photos.py (hunk 2 above)
Comment (hjheath) [inline onfido/live_photos.py]: ovewriting file is bad - fix!

**Review by peterroelants — COMMENTED** [review]
I had a couple of minor questions, remarks. If these are dealt with it looks ok to me.

File: onfido/onfido.py (hunk 3 above)
Comment (hjheath) [reply onfido/onfido.py:19]: I agree. However these make up the public API of the package and so it is outside the scope of this PR to change the global style.

Perhaps I will tidy this up at the weekend.

Comment (peterroelants) [reply onfido/onfido.py:19]: Ok, its not that important imo

**Review by peterroelants — APPROVED** [review]

**Pull request merged**